test/e2e/sched/install/install_test.go

```diff
@@ -75,9 +75,21 @@ var _ = Describe("[Scheduler] install", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			By("checking the NumaResourcesScheduler Deployment is correctly deployed")
-			deploy, err := schedutils.GetDeploymentByOwnerReference(nroSchedObj.UID)
-			Expect(err).NotTo(HaveOccurred())
-			Expect(deploy.Status.ReadyReplicas).To(BeIdenticalTo(int32(1)))
+			const DeploymentCheckTimeout = 5 * time.Minute
+			const DeploymentCheckPollPeriod = 10 * time.Second
+			Eventually(func() bool {
+				deploy, err := schedutils.GetDeploymentByOwnerReference(nroSchedObj.UID)
+				if err != nil {
+					klog.Warningf("unable to get deployment by owner reference: %v", err)
+					return false
+				}
+
+				if deploy.Status.ReadyReplicas != 1 {
```

Comment:
Let's not hardcodec constants: let's check the ready replicas are equal to the expected replicas (can't remember the field name atm)

Reply:
ok